### DISCUSSION TITLE
Build xDrip4IoS fails

### DISCUSSION THREAD

**Discussion by s187942** [JohanDegraeve/xdripswift#475] [General]
Hi all - hopefully there will be a answer to this here.

I followed the Fastlane instructions, and have had a functioning app for quite a while now - but every time the main branch was updated or a new build, then step 4 of the workflow - build xDrip failed, but it would work if I manually ran it.

Last week, with the new build, step 4 failed again, but this time running the step manually had no effect - it always fails in the Check Upstream section, specifically the Keep Alive part - this is the error message

 "Error: {"command":"git push origin HEAD","exitCode":128,"outputData":"","errorData":"remote: Permission to s187942/xdripswift.git denied to github-actions[bot].\nfatal: unable to access 'https://github.com/s187942/xdripswift.git/': The requested URL returned error: 403\n"}"

It has been suggested that I re generate a new GH_PAT - which I did and replaced the old with the new, re ran all 4 steps of the flow, and again it failed at the same place.

Would welcome any thoughts or ideas as to what to do, and why.

Many thanks

Simon

**Comment by paulplant** [comment]
Archiving as very old (and we never checked Discussions to see it)
